packages/core/entity/FileDetail/index.ts

```diff
@@ -156,62 +156,20 @@ export default class FileDetail {
         return path as string;
     }
 
-    public get cloudPath(): string {
-        //// REMOVE THIS (BACKWARDS COMPAT)
-        if (this.path.startsWith("/allen")) {
-            return this.convertAicsDrivePathToAicsS3Path(this.path);
-        }
-        ////
-
-        // AICS FMS files' paths are cloud paths
-        return this.path;
-    }
-
     public get localPath(): string | null {
-        //// REMOVE THIS (BACKWARDS COMPAT)
-        if (this.path.startsWith("/allen")) {
-            return this.path;
-        }
-        ////
-
-        if (this.getAnnotation("Cache Eviction Date")) {
-            return this.getLocalPath();
-        }
-        return null;
-    }
-
-    public getLocalPath(): string | null {
-        const localPrefix = NAS_HOST_PREFIXES[this.env];
         const cloudPrefix = `${AICS_FMS_S3_URL_PREFIX}${AICS_FMS_S3_BUCKETS[this.env]}`;
-
-        if (this.path.startsWith(cloudPrefix)) {
+        if (this.getAnnotation("Cache Eviction Date") && this.path.startsWith(cloudPrefix)) {
+            const localPrefix = NAS_HOST_PREFIXES[this.env];
             const relativePath = this.path.replace(`${cloudPrefix}`, "");
             return `${localPrefix}${relativePath}`;
         }
 
         return null;
     }
 
-    public get downloadPath(): string {
-        //// REMOVE THIS (BACKWARDS COMPAT)
-        if (this.path.startsWith("/allen")) {
-            return `http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${this.path}`;
-        }
-        ////
-
-        const localPath = this.getLocalPath();
-        // For AICS files that are available on the Vast, users can use the cloud path, but the
-        // download will be faster and not incur egress fees if we download via the local network.
```

Comment:
No more labkey download?

Reply:
Since all files have cloud paths we can just download from there now rather than use the NGINX server redirect to the Vast

Reply:
True, I am able to download that file on this branch